keepercommander/commands/enterprise.py

```diff
@@ -1335,6 +1335,42 @@ def chunk_size(self):
         return 250
 
 
+class AuditLogJsonExport(AuditLogBaseExport):
+    def __init__(self):
+        AuditLogBaseExport.__init__(self)
+
+    def default_record_title(self):
+        return 'Audit Log: JSON'
+
+    def get_properties(self, record, props):
+        filename = record.login
+        if not filename:
+            filename = input('JSON File name: ')
+            if not filename:
+                return
+            record.login = filename
+            self.store_record = True
+        props['filename'] = record.login
+
+    def convert_event(self, props, event):
+        dt = datetime.datetime.fromtimestamp(event['created'], tz=datetime.timezone.utc)
+        evt = event.copy()
+        evt.pop('id')
+        evt.pop('created')
+        evt['timestamp'] = dt.strftime('%Y-%m-%dT%H:%M:%SZ')
+        return evt
+
+    def export_events(self, props, events):
+        filename = props['filename']
+        logf = open(filename, mode='w')
+
+        try:
+            json.dump(events, logf)
+        finally:
+            logf.flush()
+            logf.close()
+
```

Comment:
export_events overrides already exported events. 
The export file contains events from the last chunk only.
Chunk is 1000 events.

Reply:
Yes, that is true. This behavior is also stated in readme. We have use case in our company, that we need only thouse incremental logs and process them with different application before sending to some system. It is much easier to deserialize JSON then parsing lets say for example syslog format. So I thought, that it may be helpful to someone else.

Reply:
Even incremental sync is going to be truncated to the last chunk if number of audit events exceeds 1000. The result JSON file will never contain more than 1000 events.
For instance: there are 5100 audit event to be downloaded. The resulting export file is going contain last 100 events. The previous 5000 events are going to be overwritten

Reply:
Probably, exporting audit log to JSON requires output file rotation similar to the one used in export to syslog

Reply:
Oh, I didn't realize that it is exporting events in chunks and that file is overwritten with every iteration. See latest commit.